packages/containers/src/ComponentForm/kit/createTriggers.js

```diff
@@ -30,6 +30,7 @@ const DEFAULT_HEADERS = {
 function noOpTrigger({ error, trigger }) {
 	// eslint-disable-next-line no-console
 	console.error(`${JSON.stringify(trigger)} failed with error ${error || '-'}`);
+	return {};
```

Comment:
this fix a bug if a trigger is called but do not exist, we should return empty data